control/available_plugin.go

```diff
@@ -555,7 +555,7 @@ func (ap *availablePlugins) streamMetrics(
 }
 
 func (ap *availablePlugins) publishMetrics(metrics []core.Metric, pluginName string, pluginVersion int, config map[string]ctypes.ConfigValue, taskID string) []error {
-	key := strings.Join([]string{plugin.PublisherPluginType.String(), pluginName, strconv.Itoa(pluginVersion)}, core.Separator)
```

Comment:
**To reviewers**:
It is only unification of the way of creating plugin_key (changed usage of Join to Sprintf) - @marcintao's suggestion. It  does not impact on the way how it works.